ui/src/tasks/components/TaskForm.tsx

```diff
@@ -139,33 +128,16 @@ export default class TaskForm extends PureComponent<Props, State> {
               offset={offset}
               cron={cron}
             />
-            {isInOverlay && (
-              <Grid.Column widthXS={Columns.Six}>
-                <Form.Element label="Output Organization">
-                  <TaskOptionsOrgDropdown
-                    orgs={orgs}
-                    selectedOrgName={toOrgName}
-                    onChangeOrgName={onChangeToOrgName}
-                  />
-                </Form.Element>
-              </Grid.Column>
-            )}
             {isInOverlay && (
               <Grid.Column widthXS={Columns.Six}>
                 <Form.Element label="Output Bucket">
-                  <GetOrgResources<Bucket>
-                    organization={this.toOrganization}
-                    fetcher={getBuckets}
-                  >
-                    {(buckets, loading) => (
-                      <TaskOptionsBucketDropdown
-                        buckets={buckets}
-                        selectedBucketName={toBucketName}
-                        onChangeBucketName={onChangeToBucketName}
-                        loading={loading}
-                      />
-                    )}
-                  </GetOrgResources>
+                  <GetResources resource={ResourceTypes.Buckets}>
+                    <TaskOptionsBucketDropdown
+                      selectedBucketName={toBucketName}
+                      onChangeBucketName={onChangeToBucketName}
+                      loading={RemoteDataState.Done}
```

Comment:
should this always be `done`? if so does the TaskOptionsBucketDropdown need the loading prop since its not rendered until its done loading from the GetResources?

Reply:
Thanks the state should be consumed in the child instead.